src/knx/ip_data_link_layer.cpp

```diff
@@ -70,9 +70,17 @@ void IpDataLinkLayer::loop()
             _platform.sendBytesUniCast(hpai.ipAddress(), hpai.ipPortNumber(), searchResponse.data(), searchResponse.totalLength());
             break;
         }
+        case SearchRequestExt:
+        {
+            // FIXME, implement (not needed atm)
+            break;
+        }
         default:
+#ifdef KNX_LOG_IP
```

Comment:
Can we not clutter the code with different LOG_THIS, LOG_THAT, LOG_SOMETHING_ELSE? If I remember correctly DEBUG should already be used in the code.  If there is more granular logic needed. There could be more levels like INFO, WARN, ERR or something.

Reply:
the logging in the stack is very basic and has no levels. Its pain in the ass to debug specific stuff when you first have to comment in 100 lines.
That was the obvious, non breaking solution.
We (@traxanos and me) plan to rework this, with log levels AND specific settings that allow you to log only what you need.